spec/models/employee_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Employee, type: :model do
```

Comment:
Since you have some public methods in the model file, you have to cover them by tests too